book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [OPTIONAL] When adding a rental for a book, it would be nice to know the person and the date of the rental. In that case, the example below will be a handy refactor code:

```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```